static/app/views/performance/cache/cacheLandingPage.tsx

```diff
@@ -194,7 +197,10 @@ export function CacheLandingPage() {
               <ModuleLayout.Full>
                 <TransactionsTable
                   data={transactionsListWithDuration}
-                  isLoading={isTransactionsListLoading || isTransactionDurationLoading}
+                  isLoading={
+                    isTransactionsListLoading ||
+                    (enableTransactionDuration && isTransactionDurationLoading)
```

Comment:
I usually do this by using `isFetching` instead of `isLoading`, which works well in this case (though it's also awkward)

```jsx

const { data, isFetching } = useBlahBlah
```

Reply:
Yup, is fetching is the way to go! thanks!!!